pkg/cmd/roachtest/testselector/snowflake_query.sql

```diff
@@ -39,9 +39,14 @@ select
   test_name,
   case when
          -- mark as selected if
-         failure_count > 0 or -- the test has failed at least once in the past "forPastDays" days
-         first_run > dateadd(DAY, ?, ts.t) or -- recently added test - test has not run for more than "firstRunOn" days
-         last_run < dateadd(DAY, ?, ts.t) or -- the test has not been run for last "lastRunOn" days
+          -- the test has failed at least once in the past "forPastDays" days
+          -- recently added test - test has not run for more than "firstRunOn" days
+          -- the test has not been run for last "lastRunOn" days
+          -- the last failure was VM preemption
```

Comment:
nit: indentation seems a bit off

Reply:
I made it that way to make it sub points